### PR TITLE
adding option to remove wrapper

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

- added option to not use wrapper all together

### CODE REVIEW

File: schemas/globalManifest.json [hunk 1]
```diff
@@ -40,6 +40,14 @@
...
 					"type": "string",
 					"description": "With this key you can change the id selector for output css variables so you can have multiple builds in one project. Default: esCssVariables"
+				},
+				"outputCssGloballyAdditionalStyles": {
+					"type": "array",
+					"description": "With this key you append additional css variables to the final inline output style tag. You can use this key to provide loading state for example. Default: []."
+				},
+				"useWrapper": {
+					"type": "boolean",
+					"description": "With this key you can change if you want to use wrapper in your project. Default: true"
```
Comment (goranalkovic-infinum) [inline schemas/globalManifest.json:50]:
```suggestion
					"description": "With this key you can change if you want to use Wrapper in your project. Default: true"
```

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]
Nitpicks 😄

**Review by volfkarlo — APPROVED** [review]
Seems good 👏

**Review by goranalkovic-infinum — APPROVED** [review]

**Review by iobrado — APPROVED** [review]

**Review by mbmjertan — APPROVED** [review]

**Review by dadadavorin — APPROVED** [review]

**Review by dingo-d — APPROVED** [review]
Would be good to have passing tests

**Pull request merged**